add relative path snippet (#6547)
fishtown-analytics/docs.getdbt.com: website/docs/reference/project-configs/asset-paths.md

## Definition
Optionally specify a custom list of directories to copy to the `target` directory as part of the `docs generate` command. This is useful for rendering images in your repository in your project documentation.

## Default

> By default, dbt will not copy any additional files as part of docs generate, i.e. `asset-paths: []`.

Reviewer: ```suggestion
By default, dbt will not copy any additional files as part of docs generate. For example, `asset-paths: []`.
```
